combat-trainer.lic

```diff
@@ -1764,6 +1764,9 @@ class TrainerProcess
 
     @combat_training_abilities_target = settings.combat_training_abilities_target
     echo("  @combat_training_abilities_target: #{@combat_training_abilities_target}") if $debug_mode_ct
+  
+    @pet_count = get_boxes(@pet_box_source).count if @skill_map.include? 'Locks'
```

Comment:
💡 What would you think about making the 'else' case explicit? Something like `@pet_count = @skill_map.include?('Locks') ? get_boxes(@pet_box_source).count : 0`

Reply:
👍